packages/kbn-securitysolution-io-ts-utils/src/non_empty_string_array/index.test.ts

```diff
@@ -8,7 +8,7 @@
 
 import { pipe } from 'fp-ts/lib/pipeable';
 import { left } from 'fp-ts/lib/Either';
-import { foldLeftRight, getPaths } from '@kbn/securitysolution-io-ts-utils';
+import { foldLeftRight, getPaths } from '../test_utils';
```

Comment:
@jbudz can you try to import from `@kbn/securitysolution-io-ts-utils/target/test_utils` please?